packages/@uppy/xhr-upload/src/index.js

```diff
@@ -124,9 +124,8 @@ module.exports = class XHRUpload extends Plugin {
     this.handleUpload = this.handleUpload.bind(this)
 
     // Simultaneous upload limiting is shared across all uploads with this plugin.
-    // __queue is for internal Uppy use only!
-    if (this.opts.__queue instanceof RateLimitedQueue) {
-      this.requests = this.opts.__queue
+    if (internalRateLimitedQueue in this.opts) {
+      this.requests = this.opts[internalRateLimitedQueue]
```

Comment:
@arturi This is to address the comment here: `XHRUpload` has to check if `__queue` property is indeed the internal one with an `instanceof` check. By switching to a `Symbol`, we can be confident it's not clashing with a user-defined `__queue` property.